docs/openapi/specification.yml

```diff
@@ -38,61 +299,191 @@ paths:
               schema:
                 $ref: '#/components/schemas/SearchResponse'
 
+  /autocomplete.json:
+    get:
+      summary: Complete a partial search query
+      description: Returns a list of suggestions for completing a user's partial search query
+      tags:
+        - Querying
+      parameters:
+        - name: q
+          in: query
+          required: true
+          description: The partial search query to autocomplete
+          schema:
+            type: string
+          example: "test"
+      responses:
+        '200':
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AutocompleteResponse'
+
+
 components:
   schemas:
     SearchResponse:
       type: object
       properties:
         results:
+          type: array
+          items:
+            $ref: '#/components/schemas/SearchResult'
+          description: The array of search results.
+        total:
+          type: integer
+          description: |-
+            The total number of items that match the search criteria.
+
+            Note that this is an approximation and is unlikely to be completely accurate even when
+            repeating the same query in quick succession.
+          example: 123
+        start:
+          type: integer
+          description: The starting offset for the returned search results.
+          example: 0
+        suggested_queries:
+          type: array
+          items:
+            $ref: '#/components/schemas/SuggestedQuery'
+          description: |-
+            Suggested queries for spelling corrections.
+
+            Note that this is an array for compatibility with v1, but will only ever have at most
+            one element (or zero if there are no meaningful completions).
+        discovery_engine_attribution_token:
+          type: string
+          description: |-
+            Attribution token from Discovery Engine.
+
+            This is used for analytics purposes to uniquely identify the search request.
+          example: >-
+            tgHwtQoLCLTQobYGEOeqt2wSJDY2Y2ZhYTBhLTAwMDAtMjIwZi04MGEwLWQ0ZjU0N2YwMjMxNCIHR0VORVJJQyp4t5KuMPn2sy3Usp0VmNa3LeuCsS2t-LMtmd6oL87mtS-0kq4w5-2ILdHmtS_C8J4Vo4CXIre3jC3ogrEtlt6oL6r4sy2b1rct3o-aIqvEii389rMtjr6dFYCymiKgibMtx8vzF4OymiKjibMt5O2ILduPmiKuxIot
```

Comment:
I trust that this is a fake token 😄

Reply:
It is indeed 😁 But in some alternate universe it may relate to something...

Reply:
TBF I might make it more obviously fake so that people/cyber scanning tools don't get confused or worried